src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs

```diff
@@ -541,7 +541,7 @@ private HashSet<int> ListUsedPrefixes(Dictionary<string, string> nsList, string
                         }
                         if (numeric)
                         {
-                            long index = long.Parse(name.Substring(prefixLength), NumberStyles.Integer, CultureInfo.InvariantCulture);
+                            long index = long.AsSpan(name.Substring(prefixLength), NumberStyles.Integer, CultureInfo.InvariantCulture);
```

Comment:
Change to: `long index = long.Parse(name.AsSpan(` ?

Reply:
Yes.  This was a typo.

Reply:
Except from that small typo, LGTM.